tools/build_all.py

```diff
@@ -65,6 +65,16 @@ def build_examples(variant):
 subprocess.run("arduino --board adafruit:nrf52:feather52832:softdevice=s132v6,debug=l0 --save-prefs", shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
 build_examples('feather52832')
 
+print('\r\n')
+print(build_separator)
+print('| {:^73} |'.format('Circuit Playground Bluefruit Express'))
+print(build_separator)
+print((build_format + '| {:5} |').format('Library', 'Example', 'Result', 'Time'))
+print(build_separator)
+subprocess.run("arduino --board adafruit:nrf52:cplaynrf52840:softdevice=s140v6,debug=l0 --save-prefs", shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+build_examples('feather52840')
```

Comment:
this should be `build_examples('cplaynrf52840')`

Reply:
nevermind, It doesn't make any differences. I will clean up the build script later :)